staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/fieldmanager.go

```diff
@@ -67,18 +67,19 @@ type Manager interface {
 // FieldManager updates the managed fields and merge applied
 // configurations.
 type FieldManager struct {
-	fieldManager Manager
+	fieldManager  Manager
+	isSubresource bool
```

Comment:
I guess, considering what I just said, "ignoreRequestObjectManagedFields" is more accurately what this changes about the logic. That name is a mouthful, maybe you can think of a better one. Personally, I prefer my fields to be named for the effect they accomplish rather than the source of the decision. That ensures that somewhere a reader will see a line like `ignoreManagedFieldsFromRequestObject = isSubresource` and it will be very clear both what we are doing and why we are doing it.

Reply:
Thanks for the tip, sounds a good idea to me - I will make the change 👍